README.md

```diff
@@ -352,27 +352,26 @@ git remote add origin https://github.com/BlackHeart-Dev/try-git.git
 - Git remote add just creates an entry in your git config that specifies a name for a particular URL. 
   You must have an existing git repo to use this.
   
-- Git clone Will physically download the files into your computer. It will take space from your computer. 
-  If the repo is 200Mb, then it will download that all and place it in the directory you cloned.
```

Comment:
I do not like shortcuting the two lines in one line, For the lines of the dimension of the same.